cupy_setup_build.py

```diff
@@ -728,7 +731,7 @@ def cythonize(extensions, arg_options):
     # Embed signatures for Sphinx documentation.
     directives['embedsignature'] = True
 
-    cythonize_option_keys = ('annotate',)
+    cythonize_option_keys = ('annotate', 'compile_time_env')
```

Comment:
This fix looks not needed anymore.

Reply:
OK I'll take it off from this PR and revive it in another (was planning to touch the nccl module using this capability).